src/tests/src/solver/optim-model-filler/test_componentFiller.cpp

```diff
@@ -18,7 +18,7 @@
  * You should have received a copy of the Mozilla Public Licence 2.0
  * along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
  */
-
+#if 0
```

Comment:
What do you want to do with these tests ?
Remove them completely or adapt them ?

Reply:
yes, work in progress